source/references.rst

```diff
@@ -8,3 +8,7 @@
 .. [ACPI] `Advanced Configuration and Power Interface (ACPI) Specification.`
 
 .. [ArmARM] `Arm® Architecture Reference Manual for the A-profile architecture`, Arm LTD.
+
+.. [OPTEECore] `OP-TEE Core <https://optee.readthedocs.io/en/latest/architecture/core.html>`
+
+.. [TFASPM] `TF-A SPM <https://trustedfirmware-a.readthedocs.io/en/latest/components/secure-partition-manager.html>`
```

Comment:
A better reference seems to be this DT binding doc:
https://trustedfirmware-a.readthedocs.io/en/latest/components/ffa-manifest-binding.html

Also, these hyperlinks don't seem to be clickable in the rendered version.

Reply:
I meant the hyperlinks within the references (see the Bibliography section). But I'm not sure if RST supports this and we can look at this later.